### PR TITLE
rejiggle layout for additional clarity

### PR DESCRIPTION
**Pull request by DomT4**
Hopefully this is useful.

Closes https://github.com/Homebrew/homebrew/issues/36970

### CODE REVIEW

**Comment by DomT4** [comment]
[What it looks like](https://domt4.github.io/).

**Comment by melodykramer** [comment]
oh yeah, that's beautiful.

File: _layouts/index.html [hunk 1]
```diff
@@ -54,74 +54,88 @@
       <div id="informations">
         <ul>
           <li>
+            <h2>How Do I Install Homebrew?</h2>
```
Comment (jaimeMF) [inline _layouts/index.html]: Shouldn't this part also be translated? (or make it optional)

Comment (mikemcquaid) [inline _layouts/index.html]: That's the big problem we have with introducing new strings, unfortunately.

Comment (jaimeMF) [inline _layouts/index.html]: It's probably not the nicest thing, but you can check if the string is available and default to the english string: https://github.com/jaimeMF/homebrew.github.io/commit/49ec3470df2631e267c9637b2709f846ffbd2964. If someone browsing in another language notices it, he can submit the translation.

**Comment by jacknagel** [comment]
I see a few formatting issues, namely:

 * the horizontal rules don't all match anymore
 * the first new item seems to be inside a `<li>` tag, but the next new header is not... seems ripe for styling issues. there is also some curious indentation changes happening, in part because of this I think
 * there is a single new `<br>`, but it's the only one on the page, probably this can be accomplished by fixing the other issues

File: _layouts/index.html (hunk 1 above)
Comment (DomT4) [inline _layouts/index.html]: Yeah, new strings are difficult on a website that has already been translated so widely. I tried to keep the two strings fairly accessible in terms of word choice but I appreciate native language support is a better solution.

**Comment by DomT4** [comment]
> the horizontal rules don't all match anymore and there are too many of them, it's noisy

Agreed. There's some spacing issues to be ironed out, which also should fix your third point in the temporary hack I stuck in place by inserting a hard break ` <br> ` to gain a little more space.

> the first new item seems to be inside a <li> tag, but the next new header is not... seems ripe for styling issues.

That was a slip on my part. Will be fixed :smile_cat:

**Comment by DomT4** [comment]
Heh, Git closed the branch because I reset against the head and then rebased against the master to get the gem updates and then pushed to the remote forgetting to specify to push only the master. That was stupid of me. Still working on this, despite that klutz move.

**Comment by DomT4** [comment]
Opened #55.

**Pull request closed without merging**